force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -252,11 +290,11 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String createAndGetBookDetails() {
         // Create a Book object with title "A Brief History of Time" and author "Stephen Hawking"
-
+        Book newBook = new Book('A Brief History of Time','Stephen Hawking');
         // Call the getBookDetails method on the book object to get the details of the book
-
+        String newBookDetails = newBook.getBookDetails();
         // Return the details of the book
-        return null; // Replace null with the variable you used to store the result
+        return newBookDetails; // Replace null with the variable you used to store the result
```

Comment:
Good work. An example of how you can simplify. The method returns a String and getBookDetails returns a String. 
So you can just return the the value of that method call. 

```
 public static String createAndGetBookDetails() {
        return new Book('A Brief History of Time', 'Stephen Hawking').getBookDetails();
    }
 ```